recipes/ms/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: ms
+  # No version numbers are supplied, so we use the last updated date in the 
+  # header of ms.c as the version number.
+  version: "2014_03_04"
+
+build:
+  number: 0
+
+source:
+  fn: ms.tar.gz
+  url: https://webshare.uchicago.edu/users/rhudson1/Public/ms.folder/ms.tar.gz
+  md5: ef693dcc424fd5293b0bea4c68953700
+
+test:
+  commands:
+    - ms 2 1 -T > /dev/null
+
+about:
+  home: http://home.uchicago.edu/rhudson1/source/mksamples.html
```

Comment:
This is a problem --- there is no license provided with the source, and no mention made of the conditions that the code is distributed under anywhere that I can find. I thought the most consistent thing here is to not provide any licencing information, since that appears to be the intention of the author.

Perhaps I should add a comment stating that no license information is available?